commandline/src/main/java/tech/pegasys/eth2signer/commandline/CommandlineParser.java

```diff
@@ -24,6 +29,7 @@
   private final Eth2SignerCommand baseCommand;
   private final PrintWriter outputWriter;
   private final PrintWriter errorWriter;
+  private final MetricCategoryConverter metricCategoryConverter = new MetricCategoryConverter();
```

Comment:
nit: does this need to be a class level variable? Or can it be constructed in registerConverters?

Reply:
done